app/src/main/java/org/techtown/soptseminar/FollowerFragment.kt

```diff
@@ -23,60 +29,99 @@ class FollowerFragment : Fragment() {
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
-        _binding = FragmentFollwerBinding.inflate(inflater, container, false)
-        initAdapter()
+        // 된다
+        homeActivity = activity as HomeActivity
+        // DataBinding 적용??
+        _binding = DataBindingUtil.inflate(inflater, R.layout.fragment_follwer, container, false)
+        if (!homeActivity.userData.isNullOrBlank()) {
+            followerAdapter = FollowerAdapter()
+            binding.rvFollower.adapter = followerAdapter
+            Log.d("UserData:", homeActivity.userData)
+            initUserInfoNetwork(homeActivity.userData)
+        }
         return binding.root
     }
 
-    private fun initAdapter() {
-        followerAdapter = FollowerAdapter {
-            val intent = Intent(requireContext(), DetailActivity::class.java).apply {
-                putExtra(GENDER, it.gender)
-                putExtra(NAME, it.name)
-                putExtra(INTRODUCE, it.introduce)
+    private fun initUserInfoNetwork(userData: String) {
+        val call: Call<List<ResponseUserInfoData>> =
+            ServiceCreator.githubApiService.getFollowingInfo(userData)
+
+        call.enqueue(object : Callback<List<ResponseUserInfoData>> {
+            @SuppressLint("NotifyDataSetChanged")
+            override fun onResponse(
+                call: Call<List<ResponseUserInfoData>>,
+                response: Response<List<ResponseUserInfoData>>
+            ) {
+                if (response.isSuccessful) {
+                    response.body()?.let { data ->
+                        Log.d("tracking?", data.toString())
+                        data.forEach {
+                            followerAdapter.followerList.add(
+                                FollowerData(
+                                    image = it.image,
+                                    name = it.name,
+                                    introduce = it.introduce ?: "NULL",
+                                )
+                            )
+                        }
+                        followerAdapter.notifyDataSetChanged()
+                    }
+                    followerAdapter.notifyDataSetChanged()
```

Comment:
```suggestion
                        followerAdapter.notifyDataSetChanged()
                    }
```
56번 라인의 response.body()?.let 에 의해서 respones.body()가 null이 아닐 경우에만 위의 코드들이 수행될 것입니다. 즉, response.isSuccessful인 상황이어도 reponse.body()가 null이면 아무런 동작을 하지 않습니다. 따라서 이중으로 notifyDataSetChanged를 수행할 필요가 없어 보입니다. :-)

Reply:
아하.. 감사합니다 이해가 됩니다